source/state_representation/test/tests/test_cartesian_state.cpp

```diff
@@ -352,7 +352,7 @@ TEST(TestPoseDistance, PositiveNos) {
   //EXPECT_TRUE(abs(d3 - 3.14159) < 1e10-3);
```

Comment:
Actually, while we're here, this test case does nothing because the EXPECTS are commented out. In addition this leads to the compiler warning that d1, d2 and d3 are unused variables. 

@buschbapti is this test syntax correct and can these EXPECTs be re-introduced?

Reply:
good point. I have just readded them with the correct syntax. It does not cover the whole cases allowed by the new syntax though